dbt_semantic_interfaces/implementations/metric.py

```diff
@@ -28,10 +28,10 @@ class PydanticMetricInputMeasure(PydanticCustomInputParser, HashableBaseModel):
     """
 
     name: str
-    filter: Optional[PydanticWhereFilterIntersection]
-    alias: Optional[str]
-    join_to_timespine: bool = False
-    fill_nulls_with: Optional[int] = None
+    filter: Optional[PydanticWhereFilterIntersection] = None
+    alias: Optional[str] = None
+    time_spine_join: bool = False
+    null_fill_value: Optional[float] = None
```

Comment:
When we tried this float caused problems when the actual value is supposed to be an int.

In a perfect world we could do `int | float` but that doesn't appear to resolve correctly and we haven't figured out how to deal with it.